### PR TITLE
Email contacts enhancements

### LINKED ISSUE #63: FEATURE: send email to HOST immediately if VISIT starts soon
(For discussion see #57)
- When a visitor clicks "SEND MY CONTACT INFO", if their visit will begin within 48 hours, the email is sent immediately to the host (an "Instant Email").
- Contacts/Visits for which an Instant Email has been sent are _not_ included in nightly emails to Hosts.

### PR DESCRIPTION
**Pull request by nellshamrell**
Fixes #63

### CODE REVIEW

**Comment by dsjoerg** [comment]
@nellshamrell my vote would be to leave this PR unmerged until the rest of #63 is done, **or** when our current primary user has an immediately upcoming campaign that needs Instant Emails.

The downside of merging this now is that hosts/organizers trying the system right now will receive multiple emails about the same visit which makes the system seem unnecessarily spammy.  However if we have an imminent organizing campaign that needs Instant Emails we would merge in this PR pronto and take the risk.

@NanKrafft sound good to you?

@nellshamrell I'm optimistic that finishing #63 will be easy compared to all the work you've already done here.

File: config/routes.rb [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
   get 'privacy_policy', to: 'policies#facebook'
 
+  resource :contacts, only: [:create]
```
Comment (dsjoerg) [inline config/routes.rb:12]: just curious, does this line create a new route?  if so, does this new route get used (and where?)   if not, why add this line?

Comment (nellshamrell) [reply config/routes.rb:12]: Hello!  I did create a new route solely for the purpose of testing the create action of the contacts controller in controller tests.  I could not successfully call the 

match "/contacts/:visit_id/:hosting_id" => "contacts#create", via: [:post]

route from rspec controller tests, despite much research and experimentation.  This route is here only for the use by rspec controller tests.

**Comment by nellshamrell** [comment]
Taking a look at finishing #63 today :)

**Comment by nellshamrell** [comment]
This is now a complete fix for #63

File: db/migrate/20161015182226_add_sent_column_to_contact.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddSentColumnToContact < ActiveRecord::Migration
+  def change
+    add_column :contacts, :sent, :datetime
```
Comment (dsjoerg) [inline db/migrate/20161015182226_add_sent_column_to_contact.rb:3]: datetime, nice!   i had expected a boolean but this is better :)

**Comment by dsjoerg** [comment]
Excellent, thanks @nellshamrell !

**Pull request merged**